src/igr-tests/igr_functions

```diff
@@ -0,0 +1,231 @@
+# Included from run-test.sh files.
+# shellcheck disable=SC2034,SC2148 # Disable some false warnings/errors.
+
+### This file contain basic variables for running Integration tests of
+### Dinit.
+
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Remove this please, we aren't using SPDX identifiers in the rest of the code base, why start doing it here?

Reply:
I thought maybe SPDX will useful. I'll remove it.